build.sbt

```diff
@@ -398,7 +377,6 @@ lazy val nscplugin =
   project
     .in(file("nscplugin"))
     .settings(mavenPublishSettings)
-    .settings(noMimaSettings)
```

Comment:
There is no binary guarantee whatsoever for the nscplugin. We should not check MiMa on this project.